webapp-django/crashstats/crashstats/forms.py

```diff
@@ -235,6 +235,16 @@ def clean_v(self):
             )
         return versions
 
+    def clean(self):
+        cleaned_data = super(DailyFormBase, self).clean()
+        if cleaned_data.get('date_end') and cleaned_data.get('date_start'):
+            # Check the invariant. Make sure the start < end.
+            if cleaned_data['date_start'] > cleaned_data['date_end']:
+                raise forms.ValidationError(
+                    'date start > date end'
```

Comment:
That's a pretty obscure error. Why not writing something a bit more explicit, like ``start date has to be lower than end date``?